guides/doc-Configuring_Load_Balancer/master.adoc

```diff
@@ -45,7 +45,11 @@ include::common/modules/proc_verifying-the-load-balancer-configuration.adoc[leve
 
 include::common/modules/con_registering-clients-to-the-load-balancer.adoc[leveloffset=+1]
 
+ifdef::orcharhino,satellite[]
 include::common/modules/proc_registering-clients-using-host-registration.adoc[leveloffset=+2]
+endif::[]
+
+include::common/modules/snip_host-registration-steps.adoc[]
```

Comment:
I'm not a fan of including a snippet in master.adoc files or assembly files because a snippet is not a standalone piece of modular content -- it lacks a heading of its own. The structure of a master.adoc or any assembly file is clearer when it consists of standalone pieces of content.

Would it be possible to include this snippet in a module (or modules) rather than master.adoc?

Reply:
Makes sense to me -> I've moved the snippet into the original file that is now orcharhino/satellite only; and added the snippet to the file above "con_registering-clients-to-the-load-balancer.adoc". the user does not know that we add a procedure to a concept file for upstream, so I deem this OK.

FYI: The original commit and change is now the last commit on my branch.